lib/matplotlib/widgets.py

```diff
@@ -809,6 +809,24 @@ def on_changed(self, func):
         """
         return self._observers.connect('changed', lambda val: func(val))
 
+    def update_range(self, vmin=None, vmax=None):
```

Comment:
```suggestion
    def set_limits(self, vmin=None, vmax=None):
```
is more in line with our API terminology.

Reply:
Done